tests/test_initiator_and_initiatorgroups.py

```diff
@@ -375,6 +377,8 @@ def test_validate_lun(setup_teardown_for_each_test):
     except exceptions.NimOSAPIError as ex:
         if "SM_invalid_arg_value" in str(ex):
             log(f"Failed as expected. Invalid param value")
+        elif "SM_not_fc_initiatorgrp" in str(ex):
+            log(f"Failed as expected. Inavlid setup")
```

Comment:
nit: Failed as expected due to invalid array setup. FC array groups are required.